Lib/fontTools/ttLib/tables/T_S_I__5.py

```diff
@@ -12,13 +12,11 @@
 class table_T_S_I__5(DefaultTable.DefaultTable):
 
 	def decompile(self, data, ttFont):
-		numGlyphs = ttFont['maxp'].numGlyphs
-		assert len(data) == 2 * numGlyphs
 		a = array.array("H")
 		a.frombytes(data)
 		if sys.byteorder != "big": a.byteswap()
 		self.glyphGrouping = {}
-		for i in range(numGlyphs):
+		for i in range(len(data) // 2):
 			self.glyphGrouping[ttFont.getGlyphName(i)] = a[i]
```

Comment:
maybe we need to add a check for the case when TSI table is too long, contains more record than are glyphs. Not sure what error you get in this case, but probably best to add a more descriptive one

Reply:
Also a logging warning may be a good idea when there are more glyphs than TSI records